apis/validator/duties/attester.yaml

```
@@ -0,0 +1,47 @@
+get:
+  tags:
+    - MinimalValidatorApi
+    - Validator
+  summary: "Get attester duties"
+  operationId: "getAttesterDuties"
+  description: "Requests the beacon node to provide a set of attestation duties,
+    which should be performed by validators, for a particular epoch.
+
+    Duties should only need to be checked once per epoch,
+    however a chain reorganization (of > MIN_SEED_LOOKAHEAD epochs) could occur,
+    resulting in a change of duties. For full safety, you should subscribe to chain reorganizations events."
```

Maybe monitor to imply polling or streaming

```suggestion
    resulting in a change of duties. For full safety, you should monitor chain reorganizations events."
```

---

We don't really have that endpoint yet, we could link to it once we add it

---

I just meant that "subscribe to" implies  a streaming endpoint (to me). Monitor seems more generic